README.md

```diff
@@ -13,12 +13,35 @@ The MLPerf Training rules are available at [training\_rules](https://github.com/
 
 The MLPerf HPC specific rules are at [hpc\_training\_rules](https://github.com/mlcommons/training_policies/blob/master/hpc_training_rules.adoc).
 
+## Compliance
+The MLPerf logging package implements logging and compliance-checking utilities
+for MLPerf benchmarks in hpc-1.1.0 branch of the MLPerf logging repository (https://github.com/mlcommons/logging/tree/hpc-1.0-branch).
+These work for the HPC v2.0 submissions as well.
```

Comment:
Actually we may have a newer branch later. This should be tentatively fine for folks who want to test things now, though